utils/HardhatDeploymentUtil.js

```diff
@@ -0,0 +1,1155 @@
+    financing,
+    onboarding,
+    daoRegistryAdapter,
+    bankAdapter,
+    nftAdapter,
+    couponOnboarding,
+    signatures,
+    tribute,
+    distribute,
+    tributeNFT,
+    erc1155Adapter,
+  } = await prepareAdapters(options);
+
+  const {
+    BankExtension,
+    NFTExtension,
+    ERC20Extension,
+    ERC1271Extension,
+    ERC1155TokenExtension,
+  } = options;
+
+  const bankAddress = await dao.getExtensionAddress(sha3("bank"));
+  const bankExtension = await (await BankExtension).attach(bankAddress);
+
+  const nftExtAddr = await dao.getExtensionAddress(sha3("nft"));
+  const nftExtension = await (await NFTExtension).attach(nftExtAddr);
+
+  const erc1271ExtAddr = await dao.getExtensionAddress(sha3("erc1271"));
+  const erc1271Extension = await (await ERC1271Extension).attach(erc1271ExtAddr);
+
+  const unitTokenExtAddr = await dao.getExtensionAddress(sha3("erc20-ext"));
+  const erc20TokenExtension = await (await ERC20Extension).attach(unitTokenExtAddr);
+
+  const erc1155TokenExtAddr = await dao.getExtensionAddress(
+    sha3("erc1155-ext")
+  );
+  const erc1155TokenExtension = await (await ERC1155TokenExtension).attach(
+    erc1155TokenExtAddr
+  );
+
+  console.log("Configure DAO");
+
+  await configureDao({
+    owner: options.owner,
+    daoFactory,
+    dao,
+    ragequit,
+    guildkick,
+    managing,
+    financing,
+    onboarding,
+    daoRegistryAdapter,
+    bankAdapter,
+    nftAdapter,
+    erc1155Adapter,
+    voting,
+    configuration,
+    couponOnboarding,
+    signatures,
+    tribute,
+    distribute,
+    tributeNFT,
+    nftAddr,
+    bankExtension,
+    nftExtension,
+    erc1271Extension,
+    erc20TokenExtension,
+    erc1155TokenExtension,
+    ...options,
+  });
+
+  return {
+    dao,
+    adapters: {
+      voting,
+      configuration,
+      ragequit,
+      guildkick,
+      managing,
+      financing,
+      onboarding,
+      daoRegistryAdapter,
+      bankAdapter,
+      nftAdapter,
+      couponOnboarding,
+      signatures,
+      tribute,
+      distribute,
+      tributeNFT,
+      erc1155Adapter,
+    },
+  };
+};
+
+const configureDao = async ({
+  owner,
+  daoFactory,
+  dao,
+  ragequit,
+  guildkick,
+  managing,
+  financing,
+  onboarding,
+  daoRegistryAdapter,
+  bankAdapter,
+  bankExtension,
+  erc1271Extension,
+  nftAdapter,
+  nftExtension,
+  erc20TokenExtension,
+  erc1155Adapter,
+  erc1155TokenExtension,
+  voting,
+  configuration,
+  couponOnboarding,
+  signatures,
+  tribute,
+  distribute,
+  tributeNFT,
+  unitPrice,
+  maxChunks,
+  nbUnits,
+  tokenAddr,
+  votingPeriod,
+  gracePeriod,
+  couponCreatorAddress,
+}) => {
+  const adapters = [];
+  if (voting) adapters.push(entryDao("voting", voting, {}));
+
+  if (configuration)
+    adapters.push(
+      entryDao("configuration", configuration, {
+        SUBMIT_PROPOSAL: true,
+        SET_CONFIGURATION: true,
+      })
+    );
+
+  if (ragequit) adapters.push(entryDao("ragequit", ragequit, {}));
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/openlawteam/tribute-contracts/pullRequest?prid=8819146)
```suggestion
  if (ragequit) {adapters.push(entryDao("ragequit", ragequit, {}));}
```